src/coreclr/debug/inc/riscv64/primitives.h

```diff
@@ -49,7 +49,7 @@ inline CORDB_ADDRESS GetPatchEndAddr(CORDB_ADDRESS patchAddr)
 
 constexpr CorDebugRegister g_JITToCorDbgReg[] =
 {
-    REGISTER_RISCV64_X0, // TODO-RISCV64-CQ: Add X0 to access DbgReg in correct order
+    (CorDebugRegister)(-1), // R0 is not using, but we need padding here for proper mapping with ICorDebugInfo::RegNum.
     REGISTER_RISCV64_RA,
```

Comment:
Also, could you please update the assert in `ConvertRegNumToCorDebugRegister` from `_ASSERTE(reg >= 0);` to `_ASSERTE(reg > 0);`.
`

Reply:
Sure, will do.

Reply:
Noti: Build fails on clang16
Error message
```
  /home/clamp/runtime/src/coreclr/debug/inc/riscv64/primitives.h:52:5: error: integer value -1 is outside the valid range of values [0, 255] for this enumeration type [-Wenum-constexpr-conversion]
      (CorDebugRegister)(-1), // X0 is zero register that is not a real register. We need padding here for proper mapping with ICorDebugInfo::RegNum.
      ^
```

Reply:
Thanks! This probably mean we should use something like `(CorDebugRegister)(255)` instead here. Initially, the sense of `-1` usage was "out of range for sure". Will fix this in next PR.